__tests__/app.test.js

```diff
@@ -37,10 +38,29 @@ describe("Testing endpoints in the app.js file", () => {
         expect(typeof endpoints).toBe("object");
       });
   });
-});
-
-describe("Error handling", () => {
-  test("GET: ", () => {});
+  test("GET: 200 - request to /api/articles/:article_id responds with an article object", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.article.article_id).toBe(1);
+        expect(response.body.article.title).toBe(
+          "Living in the shadow of a great man"
+        );
+        expect(response.body.article.topic).toBe("mitch");
+        expect(response.body.article.author).toBe("butter_bridge");
+        expect(response.body.article.body).toBe(
+          "I find this existence challenging"
+        );
+        expect(response.body.article.created_at).toBe(
+          "2020-07-09T20:11:00.000Z"
+        );
+        expect(response.body.article.votes).toBe(100);
+        expect(response.body.article.article_img_url).toBe(
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+        );
+      });
+  });
```

Comment:
Nice happy path test! But you should also have 2 error handler tests for this one